Content.Shared/Climbing/Systems/ClimbSystem.cs

```diff
@@ -251,6 +260,9 @@ private void Climb(EntityUid uid, EntityUid user, EntityUid climbable, bool sile
         if (!Resolve(climbable, ref comp, false))
             return;
 
+        if (_containers.IsEntityInContainer(uid))
+            return;
+
```

Comment:
This should be done up the callstack in canclimb or the likes not after climbing is happening.

Reply:
The ClimbingComponent field CanClimb is effectively unused as it is never altered from a true state. The code alteration could be made by subscribing to container remove as well, but if CanClimb is to be used I would recommend scoping this up from a bug fix to a ClimbSystem Rework. 

I have moved the inventory check into OnDoAfter where it will stop the call of climb. I have also observed that it is technicality unnecessary but will result in visible rollback if not checked before climbing is initiated.